src/components/accordion-card/accordion-card.md

```diff
@@ -0,0 +1,8 @@
+```js
```

Comment:
Could you please add a working example? 🙇

Reply:
⚡️